experimental/dds/tree2/src/codec/noopValidator.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { Static, TSchema } from "@sinclair/typebox";
+import { JsonValidator } from "./codec";
+
+/**
+ * A {@link JsonValidator} implementation which performs no validation and accepts all data as valid.
+ * @privateRemarks - This is useful to avoid conditional branching in codecs when the SharedTree configurer
+ * passes no validator.
```

Comment:
I think it's better to be more general than referring to ShareTree here, since nothing here is shared tree specific and this system might actually get used in other contexts some day.

```suggestion
 * @privateRemarks - Having this as an option unifies opining out of validation with selection of validators, simplifying code performing validation.
```